x-pack/plugins/console_extensions/spec/overrides/xpack.info.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_xpack": {
+    "url_params": {
+      "categories": [
+        "build",
```

Comment:
Are these supposed to be suggested as values for this param? If so, I don't see them showing up. Let me know if this is a bug and I'll file an issue.

![image](https://user-images.githubusercontent.com/1238659/41483036-6d3b24d8-708c-11e8-8556-3866c30335da.png)